docs/introduction/how-to-work-with-money.md

```diff
@@ -0,0 +1,388 @@
+]);
+```
+
+#### MoneyRange
+
+Similarly to [the Symfony `Range` constraint](https://symfony.com/doc/3.4/reference/constraints/Range.html), it validates that the amount of money is between some minimum and maximum.
+
+It has four options:
+- `min` specifies the minimum value, has to be an instance of `Money` or `null`
+- `max` specifies the maximum value, has to be an instance of `Money` or `null`
+- `minMessage` specifies the validation error message in case the entered value is less than the `min` value
+- `maxMessage` specifies the validation error message in case the entered value is greater than the `max` value
+
+At least one of the `min` and `max` options has to be provided for the constraint to make sense.
+
+```php
+use Shopsys\FormTypesBundle\MultidomainType;
+use Shopsys\FrameworkBundle\Component\Money\Money;
+use Shopsys\FrameworkBundle\Form\Constraints\MoneyRange;
+use Symfony\Component\Form\Extension\Core\Type\MoneyType;
+
+// ...
+
+$zboziFeedProductFormBuilder->add('cpc', MultidomainType::class, [
+    'label' => $this->translator->trans('Maximum price per click'),
+    'entry_type' => MoneyType::class,
+    'required' => false,
+    'entry_options' => [
+        'currency' => 'CZK',
+        'constraints' => [
+            new MoneyRange([
+                'min' => Money::create(1),
+                'max' => Money::create(500),
+            ]),
+        ],
+    ],
+]);
+```
+
+## Money in Twig Templates
+
+Instances of the `Money` class cannot be directly converted to strings.
+To be able to display a monetary value in a template you should use one of the prepared Twig filters.
+
+These filters can be used only with an instance of `Money`.
+
+### moneyFormat
+
+Formats the amount of money as a decimal number without any currency symbol.
+Optional parameters can be provided to specify the *number of decimal places* (`null` by default, meaning all), the *decimal point character* (`"."` by default) and the *separator of thousands* (`""` by default).
+
+```twig
+{# the "money" variable contains Money::create('1234.5670') #}
+
+{{ money|moneyFormat }}                  {# renders "1234.5670" #}
+{{ money|moneyFormat(0) }}               {# renders "1235" #}
+{{ money|moneyFormat(2, ',') }}          {# renders "1234,57" #}
+{{ money|moneyFormat(null, '.', ',') }}  {# renders "1,234.5670" #}
+```
+
+### price
+
+Filter `price` formats the amount of money in a localized manner.
+It uses basic frontend currency on the current domain and the current locale (language) to format it.
+
+For example, one thousand Czech crowns would be rendered as *"CZK1,000.00"* on an English domain and as *"1 000,00 Kč"* on a Czech one.
+
+All `price*` Twig filters use this format.
+They usually differ only in the currency and locale they use.
+
+### priceText
+
+Filter `priceText` formats the amount of money in a localized manner, similarly to the `price` filter.
+The only difference is that it outputs the text *"Free"* when zero amount of money is provided.
```

Comment:
```suggestion
The only difference is that it outputs the text *"Free"* (or the corresponding translation) when zero amount of money is provided.
```

Reply:
I've also added a link to the Translations article

Reply:
I'm sorry but I cannot find the link... In which file is it?

Reply:
@simara-svatopluk search for "or the corresponding translation"  in https://github.com/shopsys/shopsys/blob/ph-tl-money-6/docs/introduction/how-to-work-with-money.md

Reply:
great thanks